.github/workflows/ee-security_ci.yml

```diff
@@ -0,0 +1,15 @@
+name: WildFly ee-security Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'ee-security/**'
+      - .github/workflows/quickstart_ci.yml
+
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: ee-security
+      MICROPROFILE: false
```

Comment:
now it should instead be 
TEST_PROVISIONED_SERVER: true

Reply:
Fixed!